src/jit/lower.h

```diff
@@ -133,30 +133,32 @@ class Lowering : public Phase
 
     //----------------------------------------------------------------------
     // SetRegOptional - sets a bit to indicate to LSRA that register
-    // for a given tree node is optional for codegen purpose.  If no
+    // for a given child node is optional for codegen purpose.  If no
     // register is allocated to such a tree node, its parent node treats
-    // it as a contained memory operand during codegen.
+    // it as a contained memory operand during codegen.  This routine
+    // also updates srcRegOptional count of parent node.
```

Comment:
Yes.  Initially I have named it as srcRegOptionalCount and forgot to update it after rename. Updated.